data/projects/newcomer_metrics.md

```diff
@@ -0,0 +1,20 @@
+coala tries to be a welcoming organisation for newcomers, giving them clear pathways and as much direct assistance as needed and possible, and using automation tools to reduce the load on the core team.
+
+We need to measure our community's success in this department, in order to identify problems and constantly improve.
+We can also use metrics to gamify this process, allowing newcomers to see how they performed at the newcomer process compared to other newcomers, and giving badges to newcomers who perform particularly well, and badges to developers who spend their time assisting a newcomer through the process successfully.
+The inverse is also possible, allowing the core team to evaluate when someone has consumed too much time with too little progress, and allowing the newcomer and their peers to see cold facts that show who is taking too long to complete the process.
+
+The primary metric measured is the time period for each newcomer from first sighting on github/gitter, to the merge of a PR for a low difficulty issue that was not created by the newcomer, and a ‘suggestions requested’ review of a low difficulty PR by another community member and the review endorsed by a maintainer with a ‘+1’.(meta-reviews)
+
+This will require that issues are manually downgraded to 'difficulty/newcomer' if they were not actually the 'difficulty/low' originally expected, and
+each newcomer knows they need to have a maintainer super-review their review, and actively beg for this super-review.
```

Comment:
This isnt about "all newcomers", but I think I need to clarify this.  It is for the newcomer who has done their PRs and reviews, which is the end of the newcomer process.

This is exactly the current process in the newcomer guide, where they ask to become a 'developer'.   We *should* be meta-reviewing their code reviews then, and we *should* be pro-actively doing it.

If they beg too much, they reduce their 'score' because they are too noisy.

https://github.com/coala/coala/issues/3615 is for providing a good meta-review system.
I'd like to make a project concept for that also.
As volume of newcomers->developers increases, the need for a good meta-review system increases.